src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
+  "minecolonies:chickenherder.job.desc": "Gathers eggs, feathers, and other chicken products.",
+  "minecolonies:ranger.job.desc": "Distance fighter, deals high damage, but has low survival chance in close combat.",
+  "minecolonies:knight.job.desc": "Close-quarter fighter. High resilience, low damage output.",
+  "minecolonies:composter.job.desc": "Composts a lot of the colony waste products",
+  "minecolonies:student.job.desc": "Reads books to increase their intelligence. High intelligence allows learning other skills faster.",
+  "minecolonies:archertraining.job.desc": "Archer Trainee. Practices distance fighting.",
+  "minecolonies:combattraining.job.desc": "Knight Trainee. Practices close-quarter fighting.",
+  "minecolonies:sawmill.job.desc": "Provides the colony with most wood based items.",
+  "minecolonies:blacksmith.job.desc": "Provides the colony with most tools, armor, and weapons.",
+  "minecolonies:stonemason.job.desc": "Handles most stone based recipes.",
+  "minecolonies:stonesmeltery.job.desc": "Provides the stone variants that require smelting and charcoal",
+  "minecolonies:crusher.job.desc": "Crushes cobble to gravel, gravel to sand, and sand to clay.",
+  "minecolonies:sifter.job.desc": "Sifts through a variety of materials to find goodies.",
+  "minecolonies:florist.job.desc": "Plants a large variety of flowers for the colony.",
+  "minecolonies:enchanter.job.desc": "Gathers mana by observing others. Enchants colonist tools and creates enchanting books.",
+  "minecolonies:researcher.job.desc": "Researches the topics the player selects.",
+  "minecolonies:healer.job.desc": "Heals wounded entities and provides cures for the sick.",
+  "minecolonies:pupil.job.desc": "Sits around and learns from the material provided by the teacher.",
+  "minecolonies:teacher.job.desc": "Sits with the kids and provides material.",
+  "minecolonies:dyer.job.desc": "Handles most of the dyeing that is required in the colony.",
+  "minecolonies:glassblower.job.desc": "Provides most glass-based items.",
+  "minecolonies:mechanic.job.desc": "Handles redstone recipes and things other workers can't handle.",
+  "minecolonies:fletcher.job.desc": "Handles leather, string and wool based recipes.",
+  "minecolonies:planter.job.desc": "Provides sugar cane, cactus, and bamboo to the colony.",
+  "minecolonies:rabbitherder.job.desc": "Handles a fluffle of rabbits.",
+  "minecolonies:concretemixer.job.desc": "Creates concrete powder and actual concrete.",
+  "minecolonies:beekeeper.job.desc": "Cares for several beehives.",
+  "minecolonies:cookassistant.job.desc": "Handles food-based recipes the main cook can't handle.",
+  "minecolonies:smelter.job.desc": "Breaks ore blocks into raw ore and smelts raw ores into ingots.",
+  "minecolonies:druid.job.desc": "Support type unit. Buffs friendlies and nerfs enemies.",
+
+  "minecolonies:builder.skills.desc": "Influence the placement and breaking speed.",
```

Comment:
Some of these have "influence" and the other half have "influences" #inconsistent-ray
(I would suggest "Affects" rather than "influences" though either way)

Reply:
Also, these don't need the first "the"
E.g. "Affects ~the~ placement and breaking speed."